### PR TITLE
Scally Core -> Settings

### PR DESCRIPTION
**Pull request by chris-pearce**

### CODE REVIEW

**Comment by dansimco** [comment]
Coool lgtm :)

File: core/_settings.scss [hunk 1]
```diff
@@ -176,55 +239,55 @@ $viewport-device-width: true !default;
...
-$width-classes-at-breakpoint-non-palm: true !default;
-$width-classes-at-breakpoint-lap: false !default;
-$width-classes-at-breakpoint-lap-small: false !default;
-$width-classes-at-breakpoint-lap-small-middle: false !default;
-$width-classes-at-breakpoint-lap-large-middle: false !default;
-$width-classes-at-breakpoint-lap-large: false !default;
-$width-classes-at-breakpoint-desk: false !default;
-$width-classes-at-breakpoint-desk-small: false !default;
-$width-classes-at-breakpoint-desk-large: false !default;
+$width-classes-at-breakpoint-palm-large:              false !default;
```
Comment (AlecRaeside) [inline core/_settings.scss]: When the width classes are generated dynamically, these settings will need to be in a sass map.

**Comment by AlecRaeside** [comment]
LGTM

File: core/_settings.scss (hunk 1 above)
Comment (chris-pearce) [inline core/_settings.scss]: Yeah this will get cleaned up in a future PR, open to suggestions.

**Comment by kllevin** [comment]
LGTM

**Comment by AlecRaeside** [comment]
LGTM

**Pull request merged**